service/controllers/messages.go

```diff
@@ -624,3 +684,101 @@ func processImageResponses(result string) []messageQueryBody {
 	return queryResultArray
 
 }
+
+func processMeaning(response string) []meaningStr {
+
+	fmt.Println("this is the response")
+
+	found := false
+	
+	var subs1 = "meaning"
+	var subs2 = "example"
+	var subs3 = "subMeaning"
+	var subs4 = "subExample"
+	var subsLen1 = len(subs1)
+	var subsLen2 = len(subs2)
+	var subsLen3 = len(subs3)
+	var subsLen4 = len(subs4)
```

Comment:
[Linting Standards] use a common declaration as
```
subs1, subs2, subs3, subs4 := "meaning", "example", "subMeaning", "subExample"
```
similarily for **len** too.

Reply:
Sorry, i am changing it accordingly.